packages/react-ui-map/src/Mapbox/index.js

```diff
@@ -28,6 +49,21 @@ export default class Mapbox extends PureComponent {
       zoom: this.props.zoom,
       theme: this.props.theme
     })
+
+    map.on('load', (...args) => {
+      if(this.state.map != {map})
```

Comment:
1. I'm guessing the `{}` around `map` area accidental, either way they shouldn't be there.
2. Always favor using `!==` instead of `!=` (or alternatively `===`).
3. Are you actually using `args` for anything?  If not lets remove it.